server/publicapi/features/returning_errors.feature

```diff
@@ -2,16 +2,29 @@ Feature: Returning error on invalid request parameters
 
     Scenario: Sending an unknown parameter when retrieving items
         When we get "/items?parameter_x=foo"
-        Then we get error 422
+        Then we get error 10001
```

Comment:
Shouldn't we be returning HTTP 422 in such cases? At least that's the API specifications says.

(the same question for the other scenarios, too)